Dockerfile

```diff
@@ -44,6 +46,14 @@ RUN apk --update add bash pwgen curl \
     && rm -Rf ${EXIST_HOME}/etc/jetty/webapps/portal
 
 # adding expath packages to the autodeploy directory
+COPY update-xars.sh /tmp/update-xars.sh
+RUN chmod +x /tmp/update-xars.sh
+RUN if [[ ${UPDATE_XARS} = "true" ]] ; \
+    then /tmp/update-xars.sh ${VERSION} /tmp/xar-updates/  \
+    && rm -f ${EXIST_HOME}/autodeploy/*.xar \
+    && cp /tmp/xar-updates/*.xar ${EXIST_HOME}/autodeploy/ \
+    && rm -Rf /tmp/xar-updates/ ; \
+    else echo will not update xars ; fi 
```

Comment:
I think all this logic  is better moved into the `update-xars` script itself. Here it should only be 
```shell
RUN chmod +x /tmp/update-xars.sh && /tmp/update-xars.sh ${VERSION}
```

Reply:
good point